### DISCUSSION TITLE
Some changes to InputOptions

### DISCUSSION THREAD

**Discussion by Cheedy** [flyerhq/flutter_chat_ui#515] [Ideas & Requests]
Hello, I would like to start by saying that this feature is incredible, a great job.
I just failed to do the following points:

1) I tried to set keyboardAppearance: Brightness.dark
but I didn't find the possibility in InputOptions

2) I would like to add a microphone icon next to sendButtonVisibilityMode in order to set up a voice system but I did not find the possibility in inputOptions

Maybe I'm wrong and its features already exist, tell me if that's the case

**Comment by demchenkoalex** [comment]
Hey! I am working on a complete redesign of the package at the moment, because it became a 🍝 and is hard to maintain. At the moment, it will be faster to use `customBottomWidget` param of the `Chat` and copy the `Input` code if you want and change it, or write your own `Input`.

**Comment by Cheedy** [comment]
thanks for the answer, I will take Input in the chat params
